### ISSUE TITLE
Move image uploading functionality to backend

### ISSUE THREAD

**Issue by aminlatifi** [Giveth/giveth-2#97]
We have image uploading service on forntend side and upload images/files to pinata cloud ipfs server. Uploading feature should be moved to backend since drawbacks on current implementation (e.g. storing key and etc.)

**Comment by aminlatifi** [comment]
Image uploading is implemented by this PR: https://github.com/Giveth/impact-graph/pull/10
@brymut Two fields are added to addProject mutation. `imageUpload` to upload file, and `imageStatic` to set static path (string). I tested it using playground [Altair](https://altair.sirmuel.design/).
Also, there is one simple upload mutation for testing upload process. I suggest using it to test uploading. I tried to implement the sample uploading on frontend too, but needs some configuration to Apollo to enable Upload. The problem was empty object is received on server side when we upload image by gatsby! I pushed the incomplete implementation to this branch: https://github.com/Giveth/giveth-2/tree/f_51_upload_picture_pinata
